Assets/scripts/BattleManager.cs

```diff
@@ -126,9 +196,18 @@ private class PlayerBattleData {
 			// TODO: Add support for HOPOs
 			bool noteWasHit = false;
 			// Go through the possible notes that the player could have been trying to hit
+			GameObject[] noteObjects = GameObject.FindGameObjectsWithTag("noteObject");
 			foreach (Note n in hitNotes) { // Warning! O(n^2)
-				if (noteWasHit)
+				if (noteWasHit) {
```

Comment:
@GregoryElliott Take a look at this section of code for generating visual feedback when a note is hit. Currently the code attempts to destroy the hit note, but it clearly isn't working.